writings/original/03-chapter-2.adoc

```diff
@@ -1,6 +1,6 @@
 == Chapter 2
 
-=== An Epistle of Love to All That Are In Present Sufferings, Whether Inwardly or Outwardly.
+=== An Epistle of Love to All That Are in Present Sufferings, Whether Inwardly or Outwardly.
```

Comment:
I think it generally looks better to have headings or subheadings like this (right under a chapter number) CENTERED, instead of left justified. I think that the 
```
[.blurb]
=== 
```
is the way to go here 👍🏻

Reply:
And actually, I think that all of the headings in this chapter would look better as [.blurb] headings (which would make them centered and italicized, but not bold). 👍🏻. There are five or six headings like this in this chapter (and the same five or six also in the modernized)